### DISCUSSION TITLE
Run keycloak in Docker

### DISCUSSION THREAD

**Discussion by S-COULIBALY** [supabase/supabase#22176] [Questions]
Morning,
i have problems to run keycloak image in a docker container for production: 
i take this in superbase documentation for use keycloak to authenticate system

`docker run -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -p 8080:8080 jboss/keycloak:latest
`
i take this issiue
`docker: Cannot connect to the Docker daemon at unix:///Users/coulibalysouleymane/.docker/run/docker.sock. Is the docker daemon running?.`
 Someboby can give me the good image?

**Comment by RaphaelBossek** [comment]
Do not run Docker within WSL2 if your directory is a Windows share within WSL2.
